pkg/client/client.go

```diff
@@ -23,12 +23,10 @@ import (
 
 var logger = log.Logger("keep-ecdsa")
 
-const (
-	awaitingKeyGenerationLookback = 24 * time.Hour
-	keyGenerationTimeout          = 120 * time.Minute
-	signingTimeout                = 90 * time.Minute
-	blockConfirmations            = 12
-)
+// The default number of block confirmations the client waits until it starts the
```

Comment:
Actually... it shouldn't be `The default` but just `The number of` - we don't overwrite this value anywhere in the config file, it's the same for all the clients.